src/object/wind.cpp

```diff
@@ -109,6 +114,13 @@ Wind::collision(GameObject& other, const CollisionHit& )
     }
   }
 
+  auto badguy = dynamic_cast<BadGuy*> (&other);
+  if (badguy  &&  this->affects_badguys) {
```

Comment:
There're too many spaces around the `&&`.

Reply:
Fixed now.